tests/basic/test_left_panel.py

```
@@ -0,0 +1,387 @@
+                const text = panel.textContent.toLowerCase();
+                return text.includes('transparency') || text.includes('high') || text.includes('low');
+            }"""
+        )
+        assert transparency_visible, "Transparency category should be accessible after scrolling"
+
+
+class TestLeftPanelMobile:
+    """Test suite for left panel on mobile viewport (<992px)"""
+
+    @pytest.mark.parametrize("device_name", ALL_MOBILE_DEVICES)
+    def test_panel_hidden_by_default_on_mobile(self, mobile_page: Page, device_name: str):
```

Ruff (0.14.13)

Unused method argument: `device_name`

(ARG002)